### ISSUE TITLE
Jellyfin SSO fails behind reverse proxy

### ISSUE THREAD

**Issue by Genora51** [causefx/Organizr#1585]
###### Operating System: Ubuntu (Docker)
<hr>

##### Problem Description:
I am running Organizr and Jellyfin on Docker with a subfolder reverse proxy on `https://mysite.net`, with the Jellyfin container locally networked to `http://jellyfin:8096`. However, the Jellyfin SSO does not work. When I log in to Organizr, the SSO module sets the following for the `jellyfin_credentials` cookie (URL-decoded for clarity):

    {"Servers":[{"ManualAddress":"http://jellyfin:8096/jellyfin","Id":"...","UserId":"...","AccessToken":"..."}]}

(I have obscured the content of the tokens.)

However, the cookie should have `"ManualAddress":"https://mysite.net/jellyfin"` to make the SSO work correctly. This can be achieved by modifying the Jellyfin URL in settings from `http://jellyfin:8096/jellyfin` to `https://mysite.net/jellyfin`, but that breaks the Jellyfin Homepage item.
<hr>

In short, there should be an option in Jellyfin SSO settings to allow a _different_ URL to be set in the SSO cookie than the URL used for the homepage item to allow proper SSO on a reverse proxy setup.

**Comment by causefx** [comment]
fixed - master later today
